pep-0008.txt

```diff
@@ -639,10 +674,9 @@ When to Use Trailing Commas
 Trailing commas are usually optional, except they are mandatory when
 making a tuple of one element (and in Python 2 they have semantics for
 the ``print`` statement).  For clarity, it is recommended to surround
-the latter in (technically redundant) parentheses.
-
-Yes::
+the latter in (technically redundant) parentheses::
 
+    # Correct:
     FILES = ('setup.cfg',)
 
 OK, but confusing::
```

Comment:
I think we should add a `# Wrong:` comment to the block below.

Reply:
Should the "OK, but confusing" line be removed?

Reply:
Yeah, I think so. The "OK" part just refers to "this works", which is implicit (a style guide doesn't have to point out things that are syntax errors ;-).

Reply:
Ok